src/components/VueReactionEmoji.vue

```diff
@@ -0,0 +1,51 @@
+<template lang="pug">
```

Comment:
This is really necessary, two similar components with some differences, how about the possibility of unifying the components?

Reply:
It's possible to unify both components, however I think by unifying both components, I'm not entirely sure but I think it'll make the developer download a import a lot more usages, because the main difference essentially is that you import all images at once in `VueFeedbackReaction` and pass them as props to `VueReaction`, unlike in `VueReactionEmoji` where it **imports all the images per emoji**, so by unifying them it will be 5 times more imports than it is right now. What do you think about this?